proposals/lip-0040.md

```diff
@@ -52,6 +52,14 @@ Store keys are hashed in order to randomize the key of the leaf node, and as a c
 
 Given `B` substores in a module store and `N` entries in the substore, the module subtree will contain on average `Log(B) + Log(N)` layers. The expected number of operations needed to validate a witness from a leaf to the state root is `8 * MODULE_ID_LENGTH` (for the module ID), plus `8 * STORE_PREFIX_ID_LENGTH` (for the store prefix), plus `Log(N)` for the module subtree. Empty hashes are not included in the witness, and the resulting witness size is therefore `Log(M) + Log(B) + Log(N)`, where `M` indicates the number of modules registered in the chain.
 
+### Module ID
```

Comment:
I would use a different name here as the new identifier for modules is now the module name. I would see what is called "module ID" at the moment rather as an internal detail of the state store that generally the developer should not need to know about or care about. It should also not be used as identifier for modules in other context, i.e., methods should use the module name and not the "module ID".

From the terminology so far, we say every module has its own `module store` which is again subdivided into `substores`. From this, the most natural terminology would be to say `store prefix` or `module store prefix` for the first 4 bytes and `substore prefix` for the 2 bytes after that.

Reply:
sounds good, changed to store and substore prefixes